codersdk/updatecheck.go

```diff
@@ -6,21 +6,17 @@ import (
 	"net/http"
 )
 
-// UpdateCheckResponse contains information
-// on the latest release of Coder.
+// UpdateCheckResponse contains information on the latest release of Coder.
 type UpdateCheckResponse struct {
-	// Current is a boolean indicating whether the
-	// server version is the same as the latest.
+	// Current indicates whether the server version is the same as the latest.
 	Current bool `json:"current"`
-	// Version is the semantic version for the latest
-	// release of Coder.
+	// Version is the semantic version for the latest release of Coder.
 	Version string `json:"version"`
 	// URL to download the latest release of Coder.
 	URL string `json:"url"`
 }
 
-// UpdateCheck returns information about the latest release version of
-// Coder and whether or not the server is running the latest release.
+// UpdateCheck returns information about the latest release version of Coder and whether or not the server is running the latest release.
```

Comment:
These comments are somewhat easier to read when there are line-breaks, why do we need to make them a single line (same for the other changes)? (I personally write mine with <=80 char width but we don't enforce it, sub-80 is most prominent in the project though.)

Reply:
I admit that I changed the line length as I stick to <=120 or <=160 chars (widescreen). I believe that times with <=80 chars are already gone :)

As we fixed the bug with `<br>`, I guess that we can bring the original multi-line comment back.

Reply:
😅. For me it's not about the space, I just find long lines harder to read because my eye needs to travel more sideways.

I also admit I like it when comments align closely to the thing they describe 😆 (like this):
<img width="464" alt="image" src="https://user-images.githubusercontent.com/147409/209113959-4f6bb248-6e21-4d91-aafc-36efa815aa14.png">

Reply:
In theory, we could involve one more formatter, [golines](https://github.com/segmentio/golines), to enforce a specific line length limit, but maybe let's leave it aside from this PR.